src/explore-education-statistics-admin/src/pages/release/pre-release/PreReleasePageContainer.tsx

```diff
@@ -135,6 +136,20 @@ const PreReleasePageContainer = ({
     if (access === 'Within') {
       return (
         <>
+          <NotificationBanner
+            heading="If you have an enquiry about this release contact:"
+            title="Contact"
+          >
+            <p>
+              {`${contactTeam}: `}
+              <a
+                className='class="govuk-notification-banner__link"'
+                href={`mailto: ${contactEmail}`}
```

Comment:
I think this string interpolation is causing some whitespace in the `mailto` link: 

![image](https://user-images.githubusercontent.com/55030296/201130365-16d8224d-726d-482b-bb40-f8a2a331586a.png)

Reply:
I think the mailto will still work with the space (it does in chrome anyway), but just in case there's any other browser weirdness with it I'll get rid of the space 👍